scale_build/bootstrap/bootstrapdir.py

```diff
@@ -145,6 +149,22 @@ def after_extra_packages_installation_steps(self):
         if self.installed_packages_in_cache_changed:
             clean_packages()
 
+    def debootstrap_debian(self):
+        manifest = get_manifest()
+        run(
+            ['debootstrap'] + self.deopts + [
+                '--foreign', '--keyring', '/etc/apt/trusted.gpg.d/debian-archive-truenas-automatic.gpg',
+                manifest['debian_release'],
+                self.chroot_basedir, manifest['apt-repos']['url']
+            ]
+        )
+        for reference_file in REFERENCE_FILES:
```

Comment:
Actually... Do passwd and group exist at this moment? Will you be able to ensure that they don't contain any new lines (i.e. we don't delete any new built-in group or user by overwriting files)

Reply:
They don't exist at this moment, so it's clean that way